test/webjobs.js

```diff
@@ -1,54 +1,174 @@
 "use strict";
 
 var assert = require("assert");
+var fs = require("fs");
+var JSZip = require("jszip");
 var api = require("../")({website: process.env.WEBSITE, username: process.env.USERNAME, password: process.env.PASSWORD});
 
+function createJobZip(localPath, cb) {
+    var generateOptions = {
+        type: "nodebuffer",
+        streamFiles: true
+    };
+
+    var zip = new JSZip();
```

Comment:
I think we should use JSZip for all zip file creation. Means there is no need to rely on zip at the command line (which I didn't have).